README.md

```diff
@@ -71,12 +71,36 @@ You should _continually assess your priorities_. These are **suggested steps** b
 + https://brianclifton.com/blog/2013/09/24/google-content-experiments/
 + https://www.quora.com/What-are-the-practical-drawbacks-of-multi-armed-bandit-testing-as-applied-to-conversion-optimization
 
-**Tips to improve your testing:** 
+**Tips to improve your testing:**
 + http://marketingland.com/12-tips-to-take-your-ab-multivariate-testing-to-the-next-level-50249
 + also for interpreting your results section: http://www.optimizeandprophesize.com/jonathan_mendezs_blog/2008/05/multivariate-te.html
 + and https://econsultancy.com/blog/6740-what-your-mother-never-taught-you-about-multivariate-testing/
 
-### Possible FAQ
-+ Should I choose A/B or MVT?
-
-
+### FAQ
+#### Should I choose A/B or MVT?
+There are a few factors in whether to choose A/B or MVT testing, the main one is how long you have for the test to run before you need meaningful results.
```

Comment:
@ZooeyMiller Is this not covered in https://github.com/iteles/learn-ab-and-multivariate-testing#multivariate-vs-ab-testing ? I wonder if we're better off containing this in one place and enhancing that (pointing one repo to the other) to reduce the maintenance burden of content changes?

Reply:
I agree. The table in the learn-ab-... repo is a good explanation. Do you still think this explanation is valuable? If so I could PR to learn-ab... if not I'm fine with throwing it away!

Reply:
Yes! I think fleshing out that section in learn-ab and then referring to it from here is valuable 😊

Reply:
Awesome, will close this PR and move info over to there then.